### PR TITLE
Create Delay crawler

### PR DESCRIPTION
**Pull request by timvahlbrock**
### Description

This PR implements the delay crawler.

## Checklist

- [x] My code follows the style guidelines
- [x] I have documented my code
- [x] My changes generate no new warnings
- [x] I have linked the Issue
- [x] The Pull Request covers all the Checkpoints from the tagged Issue
- [x] I have noted the hours of work i have put in the issue

## I have edited following Parts of the project

- [ ] Frontend
- [ ] Backend
- [ ] Github
- [ ] Root

## If tests are required did you make them?

- [ ] No tests required
- [ ] Yes
- [x] No

Reason: The crawlers will require some extension of the test infrastructure to be able to test them.

### CODE REVIEW

File: schema/create_table_historic_data.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+CREATE DATABASE delays;
+
+\c delays
```
Comment (Paulomart) [inline schema/create_table_historic_data.sql:3]: what is this? :D

File: vrr_client/src/DeparturesClient.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  stopId: string;
+  limit?: number;
+}
+
+export class DeparturesClient extends VrrClientBase {
+  public async findDeparturesByStop(query: DeparturesQuery): Promise<DMTTPSchema> {
+    return this.executeFetchRequest(
+      "/static03/XML_DM_REQUEST",
+      {
+        name_dm: "de:05513:5613",
```
Comment (Paulomart) [inline vrr_client/src/DeparturesClient.ts:15]: The stopId is parameter is not use here

**Review by Paulomart — CHANGES_REQUESTED** [review]
@timvahlbrock How did you intend to call this cli (e.g. select the random stopId?)

File: schema/create_table_historic_data.sql (hunk 1 above)
Comment (timvahlbrock) [reply schema/create_table_historic_data.sql:3]: changes the selected database to "delays" instead of the default Postgres.

Comment (timvahlbrock) [reply schema/create_table_historic_data.sql:3]: One question though: Can we create multiple databases that way? I think the Postgres image only allows creation of one database.

Comment (timvahlbrock) [reply schema/create_table_historic_data.sql:3]: Or do we save the monthly data in the same database?

Comment (Paulomart) [reply schema/create_table_historic_data.sql:3]: Let's just focus on one month for now

Comment (timvahlbrock) [reply schema/create_table_historic_data.sql:3]: No I mean the timetable data that your crawler is gathering. If we create the database using compose/kubernetes we can only create one database per container. So we would need to save both of our data in the same database. But that would be fine for now in my opinion.

Comment (Paulomart) [reply schema/create_table_historic_data.sql:3]: >  So we would need to save both of our data in the same database.

👍

**Review by Paulomart — APPROVED** [review]

**Comment by timvahlbrock** [comment]
@Paulomart as you were done implementing your crawler, I added the implementation to fetch a stop by random from the DB. Could you review that again. Should be done to merge afterwards.

**Pull request merged**